### PR TITLE
Update snapshots and correctly export Terminal

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by frankban** [comment]
Self-review: it will need a follow-up anyway.

File: package.json [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+  "version": "0.2.0",
   "description": "React components for use with Juju",
   "main": "index.js",
+  "source": true,
   "scripts": {
     "lint": "eslint .",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "test": "jest"
+    "test": "jest",
+    "update-snapshot": "jest --updateSnapshot"
```
Comment (hatched) [inline package.json]: I'm not sure we want to do this, typically I update the snapshots on a case by case basis. Don't want to update the snapshot if there is indeed a failure in the code.

File: package.json [hunk 2]
```diff
@@ -48,12 +50,14 @@
   },
   "dependencies": {
     "@canonical/maraca": "^0.1.2",
+    "hoist-non-react-statics": "^3.0.1",
```
Comment (hatched) [inline package.json]: This is a little weird being here, I wonder if we shouldn't vendor it along with the modified click-outside code.

File: lib/components/basic-table/row/__snapshots__/test-row.js.snap [hunk 3]
```diff
@@ -1,21 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`BasicTableRow can display rows with expandable content 1`] = `
-<li
-  className="twelve-col basic-table__row basic-table__row--expandable basic-table__row--clickable"
+<tr
```
Comment (hatched) [inline lib/components/basic-table/row/__snapshots__/test-row.js.snap]: I'm curious as to how this were so wrong from the get go? It's like the snapshots were from a different component, maybe this has the new vanilla code but the old snapshot? Before landing this I'd like this question answered as more changes are being made to https://github.com/juju/juju-gui/pull/3831 so we'll have to keep them in sync.

Comment (huwshimi) [reply lib/components/basic-table/row/__snapshots__/test-row.js.snap]: I probably didn't update the snapshots after I copied the components over. There's no CI yet so it wasn't caught before merging.

File: package.json (hunk 1 above)
Comment (huwshimi) [reply package.json]: NPM also allows arbitrary parameters so the test command could be used instead e.g. `npm test -- --updateSnapshot`. I've been using this a lot with `jest --watch` e.g. `npm test -- --watch`.

Comment (hatched) [reply package.json]: @huwshimi cool, I didn't know that

Comment (frankban) [reply package.json]: Cool, indeed! Ok removed that line.

File: package.json (hunk 2 above)
Comment (frankban) [reply package.json]: Yes, not in the scope of this branch.

File: lib/components/basic-table/row/__snapshots__/test-row.js.snap (hunk 3 above)
Comment (frankban) [reply lib/components/basic-table/row/__snapshots__/test-row.js.snap]: Note that tests do not pass yet, but this makes us closer.

**Pull request merged**